docs/config/services/deployments/debug.js

```diff
@@ -17,9 +17,9 @@ module.exports = function debugDeployment(getVersion) {
       '../angular-sanitize.js',
       '../angular-touch.js',
       '../angular-animate.js',
-      'components/marked-' + getVersion('marked', 'node_modules', 'package.json') + '/lib/marked.js',
+      'components/marked-' + getVersion('marked') + '/lib/marked.js',
```

Comment:
Are these changes (also in the other deploy files) related to the switch to bower?